### PR TITLE
Attendee mailer

### PR DESCRIPTION
**Pull request by mmdotz**
updated show page, created invite mailer

### CODE REVIEW

File: app/views/invite_mailer/send_invite.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  <%= event.duration %>
+  <%= event.description %>
+  <%= event.room.name %>
+</p>
+
+
+<h4>Please RSVP by visiting <%= link_to 'RSVP', rsvp_url %> </h4>
+<p>
+  If you have any questions, please contact <%= user_createdevent.first_name %> <%= user_createdevent.last_name %>
+  at <%= user_createdevent.email %>
```
Comment (jah2488) [inline app/views/invite_mailer/send_invite.html.erb]: Where does `user_createdevent` come from? I am 99% certain this is going to throw an error and never actually send.

File: app/views/invite_mailer/send_invite.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<h2>You have been invited to attend this special event: </h2>
+<h3>Dear <%= @user.first_name %> <%= @user.last_name %>, </h3>
+<p>
+  You have been invited to attend this special event:
+  <%= event.start_time %>
+  <%= event.duration %>
+  <%= event.description %>
+  <%= event.room.name %>
```
Comment (jah2488) [inline app/views/invite_mailer/send_invite.html.erb]: Where does `event` come from?

File: app/views/invite_mailer/send_invite.html.erb (hunk 1 above)
Comment (jah2488) [inline app/views/invite_mailer/send_invite.html.erb]: Should probably be `user_created_event` to be in line with ruby style.

File: config/application.rb [hunk 3]
```diff
@@ -21,6 +21,7 @@ class Application < Rails::Application
     # config.i18n.default_locale = :de
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
+    config.action_mailer.default_url_options = { host: 'https://rustworks-schedlr.herokuapp.com/rsvp' }
```
Comment (jah2488) [inline config/application.rb:24]: Why does this end with `/rsvp` ? The host should only be asking for the subdomain and domain. Like this:
`config.action_mailer.default_url_options = { :host => 'yourapp.heroku.com' }`

File: config/environments/development.rb [hunk 4]
```diff
@@ -1,5 +1,7 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
+
+
```
Comment (jah2488) [inline config/environments/development.rb:4]: Lets try not to commit empty lines, leads to annoying merge conflicts :(

**Comment by jah2488** [comment]
Fix those issues and this should be good to go.

File: app/views/invite_mailer/send_invite.html.erb (hunk 2 above)
Comment (mmdotz) [inline app/views/invite_mailer/send_invite.html.erb]: this is placesaver code - we don't have a way now to segregate user that created event from user that is invited.

**Pull request merged**